lib/triggers.ts

```diff
@@ -305,20 +318,28 @@ export const getRequest = async (
 		},
 		newContractMatches || newContract,
 		options.currentDate,
+		{
+			id: options.session.actor.id,
+			type: options.session.actor.type,
+			slug: options.session.actor.slug,
+			version: options.session.actor.version,
+		},
 	);
 
 	if (!compiledTrigger) {
 		return null;
 	}
 
-	return {
+	const result = {
 		action: trigger.data.action,
 		arguments: compiledTrigger.arguments,
 		originator: trigger.id,
 		logContext: options.logContext,
 		currentDate: options.currentDate,
 		card: compiledTrigger.target,
 	};
+
+	return result;
```

Comment:
Nit: I think these changes can be reverted?

Reply:
Other than that, LGTM

Reply:
Yep, they could be, though I do prefer this approach as it makes debugging a lot easier.